content/blog/2017/2017-02-10-jenkins-datadog-plugin.adoc

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Monitor Jenkins jobs with the Datadog plugin"
+tags:
+- plugins
+- monitoring
+author: echang26
+---
+
+[NOTE]
+====
+A modified version of this article was originally posted on the
+link:https://datadoghq.com/blog/monitor-jenkins-datadog[Datadog blog].
+====
+
+
+If you're using Jenkins to continuously integrate changes into your projects, it's helpful to be able to quickly identify build failures and assess their impact on other components of your stack.
+
+Datadog's plugin helps users monitor and alert on the performance of their Jenkins builds, right alongside the rest of their infrastructure and applications.
+
+As shown in the out-of-the-box dashboard below, https://wiki.jenkins-ci.org/display/JENKINS/Datadog+Plugin[the Datadog plugin] provides a bird's-eye view of job history and trends. You can use Datadog to:
```

Comment:
fwiw we've been encouraging more and more links to plugins on plugins.jenkins.io which presents a much better user experience than crufty old Confluence :smile: